docker/README.md

```diff
@@ -0,0 +1,144 @@
+# FIWARE IoT Agent for SigFox
+
+[![FIWARE IoT Agents](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/iot-agents.svg)](https://www.fiware.org/developers/catalogue/)
+[![](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/iot-agents.svg)](https://stackoverflow.com/questions/tagged/fiware+iot)
+
+This IoT Agent is designed to be a bridge between the [Sigfox](http://www.sigfox.com/en/) callbacks protocol and the OMA
+NGSI protocol used by the [Orion Context Broker](https://github.com/telefonicaid/fiware-orion) as well as by other
+components of the FIWARE ecosystem.
+
+It is based on the [IoT Agent Node.js Library](https://github.com/telefonicaid/iotagent-node-lib). Further general
+information about the FIWARE IoT Agents framework, its architecture and the common interaction model can be found in the
+library's GitHub repository.
+
+This project is part of [FIWARE](https://www.fiware.org/). For more information check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
+
+## How to use this image
+
+The IoT Agent must be instantiated and connected to an instance of the
+[Orion Context Broker](https://fiware-orion.readthedocs.io/en/latest/), a sample `docker-compose` file can be found
+below.
+
+If the `IOTA_REGISTRY_TYPE=mongodb`, a [MongoDB](https://www.mongodb.com/) database instance is also required - the
+example below assumes that you have a `/data` directory in your hosting system in order to hold database files - please
+amend the attached volume to suit your own configuration.
+
+```yml
+version: "3.1"
+
+volumes:
+    mongodb: ~
+
+services:
+    iot-agent:
+        image: fiware/sigfox-iotagent
+        hostname: iot-agent
+        container_name: fiware-iot-agent
+        depends_on:
+            - mongodb
+        expose:
+            - "4041"
+            - "7896"
+        ports:
+            - "4041:4041"
+            - "7896:7896"
+        environment:
+            - "IOTA_CB_HOST=orion"
+            - "IOTA_CB_PORT=1026"
+            - "IOTA_NORTH_PORT=4041"
+            - "IOTA_REGISTRY_TYPE=mongodb"
+            - "IOTA_MONGO_HOST=mongo-db"
+            - "IOTA_MONGO_PORT=27017"
+            - "IOTA_MONGO_DB=iotagent-json"
```

Comment:
This DB doesn't fit the IOTA name.

I guess there is some default in the config.js file. Let's use that.

Reply:
See https://github.com/telefonicaid/sigfox-iotagent/blob/master/config.js#L146

Reply:
Fixed efc68c0